Solutions/Marain.UserNotifications.Storage.AzureStorage/Marain/UserNotifications/Storage/AzureStorage/Internal/UserNotificationTableEntity.cs

```diff
@@ -95,5 +95,22 @@ public UserNotification ToNotification(JsonSerializerSettings serializerSettings
                 new UserNotificationMetadata(correlationIds, this.ETag),
                 channelDeliveryStatuses);
         }
+
+        private static string GetHexaDecimalString(byte[] identityHash)
```

Comment:
Please can you rename to `GetHexadecimalString`.

Reply:
Yup, just pushed the changes Jon 😊